x-pack/plugins/session_view/common/constants.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const PROCESS_EVENTS_ROUTE = '/internal/session_view/process_events_route';
+export const RECENT_SESSION_ROUTE = '/internal/session_view/recent_session_route';
+export const SESSION_ENTRY_LEADERS_ROUTE = '/internal/session_view/session_entry_leaders_route';
+
+export const PROCESS_EVENTS_PER_PAGE = 1000;
```

Comment:
Is there a reason why it's 1000? Can you add a comment here if there's some performance reason or some other reason for this value?

Reply:
This has to do with the nature of the data. Being hierarchical, there are times when there are very few top level nodes, but many nested children, and there is a good chance that a page size of 100 will result in only a few line items appearing in the scrollable area. This is just a broad stroke to get a good amount of data to start with. It also plays into how our "plain text search" works. It currently only searches through content already loaded in the browser, and so this gives the user a good amount of initial data to search through before they go to load the next page. (this will eventually be improved by doing a separate backend search). I'll add a comment to explain this.

Reply:
Thank you!